app/views/search/_filters.html.erb

```diff
@@ -18,7 +18,7 @@
 
 <div class="form-group">
   <%= label_tag 'filters[format]', t('.filter_by_format'), class: 'form-label' %>
-  <%= select_tag('filters[format]', options_for_select(datafile_formats_for_select, selected_format), id: 'js_format-filter', class: 'form-control dgu-filters__filter') %>
+  <%= select_tag('filters[format]', options_for_select(datafile_formats_for_select, selected_format), id: 'format', class: 'form-control dgu-filters__filter') %>
 </div>
```

Comment:
the reason why I prefixed this id name with `js_` is to indicate that it is being used by javascript. I am afraid someone will remove this in the future, thinking it's only cosmetic.

Reply:
I don't think it's a good idea to give an id to something based on its current use. If someone removes it without checking if it's being used, they're asking for it...